content/blog/guides/2020/2020-03-25-hello-world/index.mdx

```diff
@@ -1,16 +1,173 @@
 ---
```

Comment:
I'll do the review here because I thought it might make sense to use the right tools for reviews 😂